person-service/src/main/java/gov/cdc/etldatapipeline/person/model/dto/persondetail/Phone.java

```diff
@@ -23,6 +23,8 @@ public class Phone implements ExtendPerson {
     private String cd;
     @JsonProperty("ph_tl_uid")
     private Long teleLocatorUid;
+    @JsonProperty("phone_comments")
+    private String phoneComments;
```

Comment:
Does this add an additional field `phone_comment` to the work phone output for nrt_patient when json payload with code `WP` for cd `'PSN' ` comes in?

Reply:
The input payload for patient does not have `phone_comments` field, also, output DTO - `PatientReporting` (which extends `PersonExtendedProps`) does not have it either. So even if it happens for PSN the default interface method `setPhoneComments` will be called and won't change any actual data in `PatientReporting` (and `nrt_patient`)

Reply:
We have similar case for some other fields that exist only in ProviderReporting and not exist in `PatientReporting`, `nm_degree`, e.g. There is no `nmDegree` field in `PatientReporting`, but this is always calling:
```
public class Name implements ExtendPerson {
...
    private String nmDegree;
...
    public <T extends PersonExtendedProps> T updatePerson(T person) {
        person.setLastNm(this.lastNm);
...
        person.setNmDegree(this.nmDegree);
        return person;
    }

Reply:
Got it! Thanks for the clarification!